src/BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -2733,54 +2751,20 @@ public BitmapBuffer CaptureLua()
 
 		private void IncreaseWindowSize()
 		{
-			switch (Config.TargetZoomFactors[Emulator.SystemId])
+			if (Config.TargetZoomFactors[Emulator.SystemId] < 10)

```

Comment:
Not what I meant. (This is `const`, but more precisely it's a literal, and I was suggesting a field:)
```csharp
private const int WINDOW_SCALE_MAX = 10;
// ...
ToolStripItem[] CreateWindowSizeFactorSubmenus()
{
	ToolStripItem items = new[WINDOW_SCALE_MAX];
// ...
if (Config.TargetZoomFactors[Emulator.SystemId] < WINDOW_SCALE_MAX)
```